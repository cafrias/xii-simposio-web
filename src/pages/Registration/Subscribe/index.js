// @flow
import React from 'react'
import './Subscribe.css'

import Form from './Form'

const Subscribe = () =>
  <section id="subs_form" className="subscription mobile-pad">
    <h2>Formulario de Inscripci√≥n</h2>
    <Form/>
  </section>

export default Subscribe
