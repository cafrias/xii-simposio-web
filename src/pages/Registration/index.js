// @flow
import React from 'react'

import './Registration.css'


// COMPONENTS __________________________________________________________________

import Section from '../../components/Section'
import Subscribe from './Subscribe'


// DATA ________________________________________________________________________

const DEADLINE_DATA = [
  {
    name: 'Fecha límite presentación de Resúmenes',
    weekday: 'Viernes',
    date: '21/04/2017'
  },
  {
    name: 'Fecha límite presentación de Trabajos',
    weekday: 'Viernes',
    date: '05/05/2017'
  }
]

const COST_DATA = [
  {
    name: 'Asistentes en general',
    cost: '$2.200*'
  },
  {
    name: 'Matriculados Consejo Profesional de Ciencias Económicas TDF',
    cost: '$1.600*'
  },
  {
    name: 'Docentes UNTDF',
    cost: '$1.300*'
  },
  {
    name: 'Estudiantes UNTDF',
    cost: 'Sin cargo'
  },
  {
    name: 'Estudiantes otras instituciones',
    cost: '$300**'
  }
]

function DeadLine({name, weekday, date}, key) {
  return (
    <div key={key} className="deadline">
      <h3 className="deadline__title">{name}</h3>
      <p className="table__date deadline__date">
        <span className="table__weekday">
          {weekday}
        </span>
        {date}
      </p>
    </div>
  )
}

function Cost({name, cost}, i) {
  return (
    <div key={i} className="cost">
      <h3 className="cost__title">{name}</h3>
      <p className="cost__total">
        {cost}
      </p>
    </div>
  )
}

function Registration() {
  return (
    <Section id="registration" name="Inscripción"
      className="background_white registration-container">
      <article className="deadlines">
        <h2>Presentación de trabajos</h2>
        {
          DEADLINE_DATA.map(DeadLine)
        }
      </article>
      <article className="costs">
        <h2>Costo de las matrículas</h2>
        {
          COST_DATA.map(Cost)
        }
        <p className="text-container disclaimer">
          <small>
            * Incluye material del Simposio, Cóctail de bienvenida y Cena de camaradería<br/>
            ** Incluye el material del Simposio
          </small>
        </p>
      </article>
      <article className="how-to">
        <h2>¿Cómo inscribirse?</h2>
        <p className="text-container">
          Llene la siguiente <a target="_blank"  rel="noopener noreferrer" href="https://drive.google.com/file/d/0B4-MAjalS7DYNlFjcmJCVEQtTmM/view">ficha de inscripción</a>{' '}
          y envíela a: <a href="mailto:ponencias.simposio@gmail.com">ponencias.simposio@gmail.com</a>;{' '}
          nos pondremos en contacto con usted a la brevedad.
        </p>
        <section>
          <h3>¿Cómo presentar ponencias?</h3>
          <p className="text-container">
            Para la presentación de ponencias recomendamos lea atentamente el {' '}
            <a target="_blank" rel="noopener noreferrer" href="https://drive.google.com/file/d/0B4-MAjalS7DYSGQ3WVJHRER4dXM/view">reglamento</a>.{' '}
            Cualquier consulta puede ser dirigida a: <a href="mailto:ponencias.simposio@gmail.com">ponencias.simposio@gmail.com</a>
          </p>
        </section>
      </article>
      <Subscribe/>
    </Section>
  )
}

export default Registration
