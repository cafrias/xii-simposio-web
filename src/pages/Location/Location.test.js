import React from 'react'
import { mount, shallow } from 'enzyme'

import Registration from './index'

it('should render without errors', () => {
  const wrapper = mount(<Registration/>)
  expect(wrapper).toHaveLength(1)
})

it('should match snapshot', () => {
  const wrapper = shallow(<Registration/>)
  expect(wrapper).toMatchSnapshot()
})
