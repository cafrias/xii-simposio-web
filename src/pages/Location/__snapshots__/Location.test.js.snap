// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Section
  className="background_orange location"
  id="location"
  name="Ubicación"
>
  <article
    className="tile"
  >
    <div
      className="tile__left"
    >
      <h2
        className="tile__title"
      >
        DIRECCIÓN
      </h2>
      <p
        className="text-container tile__text"
      >
        SUM de la Caja de Previsión Social de la Provincia de Tierra del Fuego,
        <br />
        Alem 2410, Ushuaia, Tierra del Fuego, Argentina.
      </p>
      <p
        className="tile__text"
      >
        <a
          href="https://www.google.com.ar/maps/place/Av.+Leandro+N.+Alem+2410,+V9410ADX+Ushuaia,+Tierra+del+Fuego/@-54.8093112,-68.3365895,16.83z/data=!4m5!3m4!1s0xbc4c233f167580ad:0xaf5184363e0591db!8m2!3d-54.8086259!4d-68.3377634?hl=en"
          target="_blank"
        >
          Ver en Google Maps
        </a>
      </p>
      <div
        className="tile__social"
      >
        <SocialLinks />
      </div>
    </div>
    <div
      className="tile__right"
    >
      <img
        alt="Mapa de la ubicacion"
        className="tile__map"
        src="map@400.png"
        srcSet="
                    map@550.png 550w,
                    map@800.png 800w,
                    map.png 1000w
                  "
      />
    </div>
  </article>
  <article
    className="tile tile--hotel"
  >
    <h2
      className="tile__title"
    >
      ALOJAMIENTO
    </h2>
    <p
      className="text-container hotel__text"
    >
      El Hotel Ushuaia ofrece una tarifa diferencial a los asistentes al Simposio, la cual podrá utilizarse informando al momento de realizar la reserva la participación en el evento:
    </p>
    <ul
      className="text-container"
    >
      <li>
        Habitación single por noche $900
      </li>
      <li>
        Habitación doble por noche $1.150
      </li>
      <li>
        Habitación triple por noche $1.300
      </li>
    </ul>
    <ul
      className="tile__social hotel__links"
    >
      <li>
        <a
          href="http://www.hotelushuaia.com.ar"
          target="_blank"
        >
          www.hotelushuaia.com.ar
        </a>
      </li>
      <li>
        <a
          href="tel:+54 2901 430671"
        >
          +54 2901 430671
        </a>
      </li>
      <li>
        <a
          href="mailto:reservas@hotelushuaia.com.ar"
        >
          reservas@hotelushuaia.com.ar
        </a>
      </li>
    </ul>
  </article>
</Section>
`;
